ProjectEureka/project-eureka-flutter
# PE-107 | Remove Dark Mode Email and Text Notifications

**what I did:**

- Removed email, text and dark mode, buttons.
- Replaced filler text with real text.

**how to test:**

1. fetch and switch to my branch
2. pull to get the latest changes
3. Make sure screen is working and button are removed.
4. Make sure you can read the text.
5. Make sure `about `button is working.
